reactioncommerce/reaction-next-starterkit
# [WIP] Fix server rendering

Resolves #182 
Impact: major
Type: fix

## Issue
Server rendering is not working

## Solution
Pass initial state data correctly to client and refactor styles to work correctly.

## Testing
1. Disable javascript in browser
2. Verify pages' HTML is rendered on the server
